editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -2955,16 +2955,14 @@ void Node3DEditorViewport::_notification(int p_what) {
 		} break;
 
 		case NOTIFICATION_PHYSICS_PROCESS: {
-			if (!update_preview_node) {
+			if (!update_preview_node || !preview_node->is_inside_tree()) {
```

Comment:
This doesn't change anything functionally, but I think it improves readability of this section that is related to this PR.